server/controllers/routes/parentProfile.js

```diff
@@ -0,0 +1,22 @@
+const jwt = require('jsonwebtoken');
+require('env2')('./config.env');
+
+const secret = process.env.SECRET_KEY;
+const { getParentProfile } = require('../../database/quieres/parentProfile');
+
+const parentPRofile = (req, res, next) => {
+  getParentProfile()
+    .then(({ rows }) => {
+      if (req.cookies.access) {
```

Comment:
same issues as in https://github.com/GSG-G7/Parent-assistent/pull/101
what's the reason for getting around the auth middleware?
+ the redirects don't seem to be correct

Reply:
Done